src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -244,6 +249,24 @@ private IOperation CreateInternal(BoundNode boundNode)
             }
         }
 
+        private IParenthesizedExpression CreateParenthesizedExpression(BoundNode boundNode)
```

Comment:
>CreateParenthesizedExpression [](start = 41, length = 29)

It feels like we should test parenthesized expressions in all possible syntax positions they can go. We should also think what expressions are interesting to cover as operands of parenthesized expressions: lambdas, method groups, tuples, etc. Not all of that have to be completed in context of this PR, I guess.

Reply:
Tracking adding additional unit tests for different syntax and error scenarios with [https://github.com/dotnet/roslyn/issues/22873](https://github.com/dotnet/roslyn/issues/22873). Let me know if you think any of this is a must for this PR.

---
In reply to: [147018917](https://github.com/dotnet/roslyn/pull/22846#discussion_r147018917) [](ancestors = 147018917)